tests/e2e/resources/installation_config/cognito-rds-s3.yaml

```diff
@@ -275,6 +275,12 @@ aws-secrets-manager:
     identifier: 'app'
     common_label: 'aws-secrets-sync'
 
+ack-sagemaker-controller:
+  installation_options:
+    kustomize: 
+      - ../../awsconfigs/common/ack-sagemaker-controller
+    helm: "/tmp/chart/sagemaker-chart"
```

Comment:
this is an installation detail and can be confusing to customers, how about remote like others?

We plan to change to add details about remote repo later

Reply:
this is required for uninstallation script 
where it will find the CRDs and delete it

Reply:
i could unzip the file to `../../chart/common/ack-controller/sagemaker-chart`
instead of `/tmp/chart/sagemaker-chart`

Reply:
in that case, we should re-download the charts during uninstallation since they are being downloaded in /tmp/charts

Reply:
unzip is also fine

Reply:
changed path to `../../chart/common/ack-controller/sagemaker-chart`